packages/Select/src/select.scss

```diff
@@ -16,11 +15,14 @@
   --ids-select-number-of-option-to-display: 4;
   --ids-select-position-ajustment-x: -1px;
   --ids-select-position-ajustment-y: -1px;
```

Comment:
` --ids-select-position-ajustment-x` and ` --ids-select-position-ajustment-y` are not used anymore